src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -626,8 +626,9 @@
    :method :post})
 
 
-(defn copy-entity-to-workspace [workspace-id]
-  {:path (str "/workspaces/" (ws-path workspace-id) "/entities/copy")
+(defn copy-entity-to-workspace [workspace-id re-link-soft-conflicts?]
+  {:path (str "/workspaces/" (ws-path workspace-id)
+              "/entities/copy?linkExistingEntities=" (true? re-link-soft-conflicts?))
```

Comment:
```cljs
(boolean re-link-soft-conflicts?)
```
since a user of this function might want to pass any non-false value to trigger this flag.